javascript/javascript3/week2/promise-visual/main.js

```diff
@@ -0,0 +1,105 @@
+async function translateOneByOne() {
+  await new Promise((resolve) => {
+    resolve(
+      moveElement(document.querySelector("ul.marks li:nth-child(1)"), {
```

Comment:
recommended to using class instead of nested child query.

Reply:
we learnt class in week3 . But can you elaborate what do you mean?

Reply:
sure, Imagen one month later, you add another li in the ul tag for another reason, then your querySelector is not valid anymore and you get a bug to find out and fix it. 

Instead, give the li a class name and use that class name in querySelector to use it.